WordPress/src/main/res/values/strings.xml

```diff
@@ -2400,7 +2400,7 @@
     <string name="my_site_blogging_prompt_card_menu_view_more">View more prompts</string>
     <string name="my_site_blogging_prompt_card_menu_skip">Skip for today</string>
     <string name="my_site_blogging_prompt_card_menu_remove">Turn off prompts</string>
-    <string name="my_site_blogging_prompt_card_attribution_dayone">From &lt;b&gt;DayOne&lt;/b&gt;</string>
+    <string name="my_site_blogging_prompt_card_attribution_dayone">From &lt;b&gt;Day One&lt;/b&gt;</string>
```

Comment:
It occurred to me that the `dayone` `name` might should be renamed to `day_one`, but I avoided doing so to limit the surface area of this change and avoid unintended regressions. I'm open to refactoring the `name` if it is deemed worthwhile.

Reply:
Also, my understanding is that string files for other languages are automatically generated and I should not update them manually. Is that correct?

Reply:
@thomashorta I refactored the changes to accommodate a new translation string. This is ready for another review.